testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -1424,49 +1413,48 @@ public void testInvalidSearch() throws IOException {
     r = RestUtils.get(authToken, collectionPath + "/cool-search-id?where={\"a\": true}}", 400);
     assertThat(r)
         .isEqualTo(
-            "{\"description\":\"Search entry for field a was expecting a JSON object as input.\",\"code\":400}");
+            "{\"description\":\"Search was expecting a JSON object as input.\",\"code\":400}");
 
     r =
         RestUtils.get(
             authToken,
             collectionPath + "/cool-search-id?where={\"a\": {\"$exists\": false}}}",
             400);
     assertThat(r)
-        .isEqualTo(
-            "{\"description\":\"The operation $exists only supports the value `true`\",\"code\":400}");
+        .isEqualTo("{\"description\":\"$exists only supports the value `true`\",\"code\":400}");
 
     r =
         RestUtils.get(
             authToken, collectionPath + "/cool-search-id?where={\"a\": {\"exists\": true}}}", 400);
-    assertThat(r).startsWith("{\"description\":\"Invalid operator: exists, valid operators are:");
+    assertThat(r).startsWith("{\"description\":\"Operation exists is not supported.");
```

Comment:
I'd quote the invalid operator for clarity. 

Also, was there any particular reason for removing valid operators from this message?

Reply:
This was changed in the error messages refactoring, now we hit that new path.. Wanna update to the old style of the error message?

Reply:
The new message is fine from my POV. I meant quoting `"exists"` in this case, otherwise it makes parsing the English text harder, IMHO :)

Reply:
Done.